src/app/services/user-data.service.ts

```diff
@@ -4,19 +4,22 @@ import { User } from '../models/User';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
 import * as moment from 'moment';
+import { TaskItem } from '../models/TaskItem';
 
 @Injectable({
   providedIn: 'root'
 })
 export class UserDataService {
 
-  testTaskList: string[];
+  private currentUserKey = 'currentUser';
+  private dateFormat = "DD-MM-YYYY";
+  testTaskList: Array<TaskItem>;
```

Comment:
Please add accesses modifier here like public or private

Reply:
Fix is done.